design/clusters.go

```diff
@@ -13,6 +13,36 @@ var clusterList = JSONList(
 	nil,
 	nil)
 
+// singleCluster represents a single cluster object
+var createCluster = JSONSingle(
+	"Cluster",
+	"Holds the response to a single cluster request",
```

Comment:
it's not response,  holds the data attributes to create cluster

Reply:
ok, let me fix that

Reply:
addressed in 59369a0